### ISSUE TITLE
Permanent filter list for all platforms

### ISSUE THREAD

**Issue by 2-m** [AHAAAAAAA/PokemonGo-Map#1476]
It's a bit tedious to constantly have to add Pokemon back to the filtered list whether you're on the web interface or are viewing the web interface on mobile. Can we have some sort of way to permanently filter certain Pokemon? I liked when -o was used, it allowed me to filter out everything but the rares

**Comment by cNyhuus** [comment]
I agree, it would be nice to server side, tell it to leave out the "junk".
There was a -i switch earlier, but that was removed, when the interface came with feature to hide pokemons from the map.

**Comment by 2-m** [comment]
@cNyhuus I agree, definitely needs to be server side

**Comment by Maxsky5** [comment]
+1
